iroh-resolver/src/content_loader.rs

```diff
@@ -24,6 +24,13 @@ pub trait ContentLoader: Sync + Send + std::fmt::Debug + Clone + 'static {
     async fn stop_session(&self, ctx: ContextId) -> Result<()>;
     /// Checks if the given cid is present in the local storage.
     async fn has_cid(&self, cid: &Cid) -> Result<bool>;
+    /// Store some content
+    async fn store_file<T: tokio::io::AsyncRead + 'static + std::marker::Send>(
+        &self,
+        _content: T,
+    ) -> Result<cid::Cid, anyhow::Error> {
+        unimplemented!()
```

Comment:
This is still unimplemented. Should we allow this?
On a separate note, feels wrong to be placed in the content loader. Unsure if the content loader is evolving to be more (and whether it should) or if we should move this out and have it be handled by the gateway client and not the resolver.